lib/crawler.js

```diff
@@ -751,12 +938,11 @@ Crawler.prototype.queueURL = function(url, queueItem) {
     }
 
     // Pass this URL past fetch conditions to ensure the user thinks it's valid
-    var fetchDenied = false;
-    fetchDenied = crawler._fetchConditions.reduce(function(prev, callback) {
+    var fetchDenied = crawler._fetchConditions.reduce(function(prev, callback) {
         return prev || !callback(parsedURL);
     }, false);
 
-    if (fetchDenied) {
+    if (fetchDenied || crawler._robotsTxtFetchCondition(parsedURL)) {
```

Comment:
My thought here was that by not placing this in the fetch conditions array, it would be less prone to being accidentally removed by the developer, but I don't know how much of a concern that is. But it would be more elegant to place it in the fetch conditions array, I agree. Which do you prefer?